homeassistant/components/powerwall/const.py

```diff
@@ -9,6 +9,8 @@
 UPDATE_INTERVAL = 30
 
 ATTR_FREQUENCY = "frequency"
+ATTR_ENERGY_EXPORTED = "energy_exported_(in_kW)"
+ATTR_ENERGY_IMPORTED = "energy_imported_(in_kW)"
```

Comment:
Looks like this is unrelated

Reply:
Yep, not sure where that came from but can be deleted